core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/jdbc/JDBCUserStoreManager.java

```diff
@@ -3780,11 +3784,11 @@ public long doCountUsersWithClaims(String claimUri, String value) throws UserSto
             }
 
         } catch (SQLException e) {
-            String msg = "Error while executing the SQL " + sqlStmt;
+            String msg = ERROR_WHILE_EXECUTING_THE_SQL;
```

Comment:
I don't think we need a constant for this. This string is used in only one place and having a constant will make it less readable.